### PR TITLE
Display consent status on consenst card

### PR DESCRIPTION
**Pull request by misaka**
image

image

### CODE REVIEW

File: app/assets/stylesheets/application.sass.scss [hunk 1]
```diff
@@ -174,5 +174,16 @@ $color_app-pale-blue: #ccdff1;
...
 
 .app-status {
+  align-items: center;
+  display: inline-flex;
   font-weight: bold;
+  gap: 4px;
+}
+
+.app-status--green {
+  color: #007f3b;
```
Comment (tvararu) [inline app/assets/stylesheets/application.sass.scss:184]: Is there an `nhsuk-color` for this?

Comment (misaka) [reply app/assets/stylesheets/application.sass.scss]: Yes! Thanks, nice catch!

Commit amended.

Comment (paulrobertlloyd) [reply app/assets/stylesheets/application.sass.scss]: See: https://github.com/nhsuk/manage-childrens-vaccinations-prototype/blob/654cf2ecff49fb1acc8a704921abb119cf96c88f/app/assets/stylesheets/application.scss#L120-L137

Comment (misaka) [reply app/assets/stylesheets/application.sass.scss]: Amending.

**Review by tvararu — APPROVED** [review]

**Pull request merged**